docs/src/main/asciidoc/spring-data-jpa-guide.adoc

```diff
@@ -0,0 +1,592 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc
+////
+= Quarkus - Using our Spring Data JPA compatibility layer
+
+include::./attributes.adoc[]
+
+While you are encouraged to use Hibernate Panache for Relational Database access, Quarkus provides a compatibility layer for
+Spring Data JPA repositories in the form of the `spring-data-jpa` extension.
+
+== Prerequisites
+
+To complete this guide, you need:
+
+* less than 15 minutes
+* an IDE
+* JDK 1.8+ installed with `JAVA_HOME` configured appropriately
+* Apache Maven 3.5.3+
+
+== Solution
+
+We recommend that you follow the instructions in the next sections and create the application step by step.
+However, you can go right to the completed example.
+
+Clone the Git repository: `git clone {quickstarts-clone-url}`, or download an {quickstarts-archive-url}[archive].
+
+The solution is located in the `using-spring-data-jpa` directory.
+
+== Creating the Maven project
+
+First, we need a new project. Create a new project with the following command:
+
+[source,shell,subs=attributes+]
+----
+mvn io.quarkus:quarkus-maven-plugin:{quarkus-version}:create \
+    -DprojectGroupId=org.acme \
+    -DprojectArtifactId=using-spring-data \
+    -DclassName="org.acme.spring.data.jpa.FruitResource" \
+    -Dpath="/greeting" \
+    -Dextensions="spring-data-jpa,jaxrs-json,quarkus-jdbc-postgresql"
```

Comment:
```suggestion
    -Dextensions="spring-data-jpa,jaxrs-json,quarkus-jdbc-postgresql"
```
I would point to the real extension for `jaxrs-json` because we could imagine having the tag on both the JSON-B and Jackson extensions and then it would fail.

Reply:
Makes sense. FWIW, I lifted this from the Panache guide so I guess it would need to be changed there as well